tests/sample/elements/layers/test_layer_area_per_molecule.py

```diff
@@ -13,13 +13,13 @@ class TestLayerAreaPerMolecule(unittest.TestCase):
     def test_default(self):
         p = LayerAreaPerMolecule.default()
         assert p.molecular_formula == 'C10H18NO8P'
-        assert p._area_per_molecule.raw_value == 48.2
-        assert str(p._area_per_molecule.unit) == 'angstrom ** 2'
-        assert p._area_per_molecule.fixed is True
+        assert p.area_per_molecule.raw_value == 48.2
+        assert str(p.area_per_molecule.unit) == 'angstrom ** 2'
```

Comment:
Not a comment on your work here, but isn't it better to do the assertions against an instance of the constructed scipp unit rather than strings? It wouldn't catch errors if the unit system iteslf broke (but that isn't the remit of these tests), but it would stop you having to rewrite alot of things if someone decided to make "angstrom" into "Angstrom".

Reply:
I like the idea and wanted to change the tests accordingly.  However, the unit is set and handled in EasyCore and it is not a scipp class.  I don't want to go down a rabbit hole to fix this and will leave it as is.